src/recenter/directive.js

```diff
@@ -1,7 +1,15 @@
-goog.provide('ngeo.recenterDirective');
+goog.provide('ngeo.recenter.directive');
 
 goog.require('ngeo');
 
+/**
+ * @type {!angular.Module}
+ */
+ngeo.recenter.directive = angular.module('ngeoRecenter', []);
+
+const ngeoModule = ngeo.module;
```

Comment:
I understand why I have to create this const, but not why it was not required in the gmf.search.component...

Reply:
@ger-benjamin, no it is not correct to use this const here: you can not create a local constant, in a `goog.provide` file.
Do you have an error when using `ngeo.module` directly?